config/templates/cluster.yaml

```diff
@@ -475,6 +475,11 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 #     # See https://github.com/coreos/kube-aws/issues/230 for more information.
 #     rbac:
 #       enabled: true
+#     authentication:
```

Comment:
I agree, i've added a link to Kubernetes' documentation about webhook token authentication